### DISCUSSION TITLE
Get In Touch With Maintainer: Entreprise Collaboration

### DISCUSSION THREAD

**Discussion by saniyusuf** [florisboard/florisboard#1922] [Feedback]
Hi There, 
I would like to get in touch with the Maintainer of this amazing project. I realised that there are no direct contact details. I have an amazing opportunity to collaborate and integrate this on an enterprise level. Feel free to reach out to me via LinkedIn https://www.linkedin.com/in/saniyusuf .

**Comment by patrickgold** [comment]
Hello Mr. Yusuf,

I am the lead maintainer and developer of this project. As I don't have a LinkedIn account I cannot contact you directly, so feel free to email me at florisboard@patrickgold.dev (public address which visible in the GitHub FlorisBoard org and repo community guidelines) and we can have a chat about your idea. Or we can discuss it here in this thread too if you want.
